examples/with-next/pages/_app.tsx

```diff
@@ -0,0 +1,76 @@
+import "../styles/global.css";
+import "@rainbow-me/rainbowkit/styles.css";
+import type { AppProps } from "next/app";
+import { useRouter } from "next/router";
+
+import {
+  RainbowKitProvider,
+  getDefaultWallets,
+  Locale,
+  getDefaultConfig,
+} from "@rainbow-me/rainbowkit";
+import {
+  argentWallet,
+  trustWallet,
+  ledgerWallet,
+} from "@rainbow-me/rainbowkit/wallets";
+import { WagmiProvider } from "wagmi";
+import {
+  arbitrum,
+  base,
+  mainnet,
+  optimism,
+  polygon,
+  sepolia,
+  zora,
+} from "wagmi/chains";
+import { QueryClient, QueryClientProvider } from "@tanstack/react-query";
+
+const projectId = "YOUR_PROJECT_ID";
+
+const appName = "Rainbowkit Demo";
+
+const demoAppInfo = {
+  appName: "Rainbowkit Demo",
+};
```

Comment:
@KosmosKey I think we should remove reference to `demoAppInfo` since we want users to pass the `appName` to `getDefaultConfig` now. Let's also add `appName` in-line after projectId cleanliness